libs/blocks/modal/modal.css

```diff
@@ -220,12 +220,37 @@
   overflow: hidden;
 }
 
+.dialog-modal.tall-video {
+  line-height: 0;
+  width: fit-content;
+}
+
+.dialog-modal.tall-video .milo-video {
+  aspect-ratio: var(--aspect-ratio-tall);
+  height: auto;
+  padding-bottom: 0;
+  min-width: 90vw;
+}
+
 @media (min-width: 600px) {
   .dialog-modal {
     max-width: 80vw;
     width: fit-content;
   }
 
+  .dialog-modal.tall-video {
+    --modal-width-var: 50vw;
+
+    max-width: var(--modal-width-var);
+    padding: var(--spacing-xl);
+  }
+
+  .dialog-modal.tall-video .milo-video {
+    --modal-width-var: 50vw;
```

Comment:
Already declared on L242, shouldn't be required here as well.

Reply:
We may need to circle back to this for a better plan. I don't have specs or answers for all of these questions right now and the feature is an opt in feature.

The main use case and priority right now is phone, along with this update being in production next week.

Reply:
Then let's just remove the duplicate `--modal-width-var` declaration and we can follow-up with any enhancements in the future.

Reply:
@overmyheadandbody, updated. Thanks for the help.